libmscore/synthesizerstate.h

```diff
@@ -86,8 +86,14 @@ class SynthesizerState : public std::list<SynthesizerGroup>
 
 static SynthesizerState defaultState = {
     { "master", {
-          { 0, "Zita1" },
-          { 2, "0.1" },
+#ifdef ZITA_REVERB
+          {
+              0, "Zita1"
+          },
+#endif
+          {
+              2, "0.1"
+          },
           { 3, "440" },
           { 4, "1" },
           { 5, "1" }
```

Comment:
Why only change the first two and not all others?

Reply:
Because the code style check only wanted those. IMHO bad enough...

Reply:
This is the problem of introducing automatic check. Do we still get to vote for its expulsion from MuseScore? :)

Reply:
It comes with a price tag for sure...